app/backend/src/couchers/servicers/threads.py

```diff
@@ -32,11 +33,15 @@ def total_num_responses(database_id):
     """
     with session_scope() as session:
         return (
-            session.query(func.count(Comment.id)).filter(Comment.thread_id == database_id).scalar()
-            + session.query(func.count(Reply.id))
-            .join(Comment, Comment.id == Reply.comment_id)
-            .filter(Comment.thread_id == database_id)
-            .scalar()
+            session.execute(
+                select(func.count()).select_from(Comment).where(Comment.thread_id == database_id)
+            ).scalar_one()
+            + session.execute(
+                select(func.count())
+                .select_from(Reply)
```

Comment:
Is a `select_from` the recommended way to do a `count`? Is it just shorthand for a subquery? Just wondering.

Reply:
This is their recommended format for count. Here's one place they mention it, though I definitely saw it somewhere else as well:

https://github.com/sqlalchemy/sqlalchemy/discussions/6334#:~:text=Correct%20-%20use%20select(func.count()).select_from(Pets)%20for%20that.

Reply:
I believe this is where I first got it:

https://stackoverflow.com/questions/65754023/what-is-the-equivalent-of-query-count-in-the-sqlalchemy-1-4-orm

Reply:
Thanks!